ga4gh/backend.py

```diff
@@ -208,6 +265,7 @@ def __init__(self):
         self._referenceSetIdMap = {}
         self._referenceSetNameMap = {}
         self._referenceSetIds = []
+        self._ontologies = {}
```

Comment:
We should probably have _ontologyIdMap, _ontologyNameMap and _ontologyIds here for consistency.